src/app/services/theme/theme.service.ts

```diff
@@ -102,6 +110,42 @@ export class ThemeService {
     }
   }
 
+  // We are overriding bootstrap at runtime because bootstrap uses a custom SCSS RGB() function to extract red, green
+  // blue values from a color. This does not work with our --baw-highlight variable because we do not know the value at
+  // compile time.
+  // we override the bootstrap variables with our own rgb() function at runtime when we know the value of baw-highlight
+  // TODO: We can replace this with the CSS rgb() function when it is supported by all browsers
+  // https://caniuse.com/mdn-css_types_color_rgb_relative_syntax
+  private overrideBootstrapColors(color: string) {
+    const targetColor = hsl(color);
+
+    bootstrapVariables.forEach(([variable, lightnessAdjustment]) => {
+      const hslColor = targetColor
+        .copy({ l: targetColor.l + lightnessAdjustment })
+        .clamp();
+
+      const rgbColor = rgb(hslColor);
+
+      if (!this.isServer) {
+        this.setBootstrapColorInBrowser(variable, rgbColor);
+      }
```

Comment:
explain why we're not doing this in SSR?

Reply:
I was under the impression that we didn't have access to the DOM in SSR

However, I investigated a bit further, and it appears that SSR has access to the DOM object model. Therefore, we can set these values server side